lib/ReactViews/Workbench/Controls/Legend.tsx

```diff
@@ -186,7 +186,7 @@ export default class Legend extends React.Component<{
         !imageUrl && legendItem.addSpacingAbove ? "1px solid black" : undefined
     };
 
-    if (!imageUrl && legendItem.outlineColor) {
+    if (!imageUrl && legendItem.outlineColor && legendItem.outlineWidth) {
```

Comment:
Are there cases where `legendItm.outlineWidth` is not defined but we still want a border? Just want to make sure that this check hasnt just been added to avoid a ts error and forgotten, I would do that!